test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -1621,15 +1621,20 @@ public void Ctor_KeepsCustomTransportOnOptions()
     [Fact]
     public void Ctor_WrapsCustomTransportWhenCachePathOnOptions()
     {
+        // Arrange
         _fixture.SentryOptions.Dsn = ValidDsn;
         _fixture.SentryOptions.Transport = new FakeTransport();
         using var cacheDirectory = new TempDirectory();
         _fixture.SentryOptions.CacheDirectoryPath = cacheDirectory.Path;
 
+        // Act
         using var sut = new SentryClient(_fixture.SentryOptions);
 
+        // Assert
         var cachingTransport = Assert.IsType<CachingTransport>(_fixture.SentryOptions.Transport);
         _ = Assert.IsType<FakeTransport>(cachingTransport.InnerTransport);
+
+        cachingTransport.Dispose(); // Release cache lock so that the cacheDirectory can be removed
```

Comment:
question: don't we need to do this on a Test-Class level instead, so that it applies to every Test-Method?